austrakka/utils/options.py

```diff
@@ -25,6 +25,28 @@ def opt_abbrev(**attrs: t.Any):
     )
 
 
+def opt_curr_owner(**attrs: t.Any):
+    defaults = {
+        'required': True,
+        'help': 'Abbreviated name of current owning organisation.'}
+    return create_option(
+        "-co",
+        "--curr-owner",
+        **{**defaults, **attrs}
```

Comment:
@viatunimelb do we _have_ to make them specify the current owner? I understand why the endpoint would like them to, but from a naive user's point of view the reaction will be "you already know that". Can we get away without it?

Reply:
I was thinking that too but decided to go with an explicit value because so many of the sample commands operate on samples belonging to different owners. Also, echoing the command becomes self-documenting - you don't need to figure out after the fact what the owning org was?